idaes_examples/notebooks/docs/surrogates/SCO2_example/ALAMO/SCO2_properties_alamo_surrogate_embedding.ipynb

```diff
@@ -0,0 +1,461 @@
+    "                    Tflag[k] = True\n",
+    "                else:\n",
+    "                    Tflag[k] = False\n",
+    "                    if state_args is None:\n",
+    "                        blk[k].temperature.fix()\n",
+    "                    else:\n",
+    "                        blk[k].temperature.fix(state_args[\"temperature\"])\n",
+    "\n",
+    "            # If input block, return flags, else release state\n",
+    "            flags = {\"Fcflag\": Fcflag, \"Pflag\": Pflag,\n",
+    "                     \"Tflag\": Tflag}\n",
+    "\n",
+    "        else:\n",
+    "            # Check when the state vars are fixed already result in dof 0\n",
+    "            for k in blk.keys():\n",
+    "                if degrees_of_freedom(blk[k]) != 0:\n",
+    "                    raise Exception(\"State vars fixed but degrees of freedom \"\n",
+    "                                    \"for state block is not zero during \"\n",
+    "                                    \"initialization.\")\n",
+    "\n",
+    "        if state_vars_fixed is False:\n",
+    "            if hold_state is True:\n",
+    "                return flags\n",
+    "            else:\n",
+    "                blk.release_state(flags)\n",
+    "\n",
+    "    def release_state(blk, flags, outlvl=0):\n",
+    "        '''\n",
+    "        Method to relase state variables fixed during initialisation.\n",
+    "\n",
+    "        Keyword Arguments:\n",
+    "            flags : dict containing information of which state variables\n",
+    "                    were fixed during initialization, and should now be\n",
+    "                    unfixed. This dict is returned by initialize if\n",
+    "                    hold_state=True.\n",
+    "            outlvl : sets output level of of logging\n",
+    "        '''\n",
+    "        if flags is None:\n",
+    "            return\n",
+    "\n",
+    "        # Unfix state variables\n",
+    "        for k in blk.keys():\n",
+    "            if flags['Fcflag'][k] is False:\n",
+    "                blk[k].flow_mol.unfix()\n",
+    "            if flags['Pflag'][k] is False:\n",
+    "                blk[k].pressure.unfix()\n",
+    "            if flags['Tflag'][k] is False:\n",
+    "                blk[k].temperature.unfix()\n",
+    "\n",
+    "        if outlvl > 0:\n",
+    "            if outlvl > 0:\n",
+    "                _log.info('{} State Released.'.format(blk.name))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we have our property package ready for being used in the flowsheet for optimization. We shall see that in the next part of this tutorial, SCO2_flowsheet_alamo_surrogate.ipynb. To learn in detail about making a custom property package, one should go through [Property Package Example](https://github.com/IDAES/examples/blob/main/idaes_examples/notebooks/docs/properties/custom/custom_physical_property_packages.ipynb). "
```

Comment:
Is there a way to link it so that "SCO2_flowsheet_alamo_surrogate.ipynb" is a hyperlink to that notebook? If there is, it would be good to do so in the other notebooks where applicable.